src/components/contract-explorer-url.tsx

```diff
@@ -33,12 +35,7 @@ const ContractExplorerUrl: React.FC<{
     address: contractAddress
   })}#code`
   return (
-    <StyledLink
-      href={url}
-      style={{ textDecoration: 'underline', color: '#4d00b473' }}
-      target="_blank"
-      rel="noopener noreferrer"
-    >
+    <StyledLink href={url} target="_blank">
```

Comment:
We are missing the `rel="noopener noreferrer"`  here
```suggestion
    <StyledLink href={url} target="_blank" rel="noopener noreferrer">
```